packages/shared/components/Text.svelte

```diff
@@ -66,6 +67,7 @@
         class:disabled
         class:highlighted
         class:error>
+        {#if value}{value}{/if}
```

Comment:
@nicole-obrien what is `value` for?

Reply:
Honestly, can't remember why I was trying this. I will clean this up and then resolve after the commit.